### PR TITLE
change mind about ec2 type and count

### PR DESCRIPTION
**Pull request by pseudonative**
Trigger this job

### CODE REVIEW

File: terraform/ec2-instance.tf [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-#     Service = "CICD"
-#     Environment = "GithubActionsWorkflow"
-#   }
-#   provider = aws.us-east-1
-# }
+resource "aws_instance" "myec2vm" {
+  ami           = data.aws_ami.amzlinux2.id
+#   instance_type = var.instance_type
+#   instance_type = var.instance_type_list[1]
+  instance_type = var.instance_type_map["dev"]
```
Comment (shoshone-dev) [inline terraform/ec2-instance.tf]: We want a type QA instance

File: terraform/ec2-instance.tf [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
+  user_data     = file("${path.module}/app1-install.sh")
+  key_name      = var.instance_keypair
+  vpc_security_group_ids = [
+    aws_security_group.vpc-ssh.id,
+    aws_security_group.vpc-web.id
+  ]
+  count = 1
+  tags = {
+    "Name" = "${var.Name}_flow_${count.index}"
```
Comment (shoshone-dev) [inline terraform/ec2-instance.tf]: name QA

File: terraform/ec2-instance.tf [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
+#   instance_type = var.instance_type
+#   instance_type = var.instance_type_list[1]
+  instance_type = var.instance_type_map["dev"]
+  user_data     = file("${path.module}/app1-install.sh")
+  key_name      = var.instance_keypair
+  vpc_security_group_ids = [
+    aws_security_group.vpc-ssh.id,
+    aws_security_group.vpc-web.id
+  ]
+  count = 1
```
Comment (shoshone-dev) [inline terraform/ec2-instance.tf]: We need a count of 3

**Review by shoshone-dev — CHANGES_REQUESTED** [review]

File: terraform/ec2-instance.tf (hunk 1 above)
Comment (pseudonative) [reply terraform/ec2-instance.tf]: instance_type = var.instance_type_map["qa"]

File: terraform/ec2-instance.tf (hunk 2 above)
Comment (pseudonative) [reply terraform/ec2-instance.tf]: "Name" = "${var.Name}_QA_${count.index}"

File: terraform/ec2-instance.tf (hunk 3 above)
Comment (pseudonative) [reply terraform/ec2-instance.tf]: count = 3

**Review by pseudonative — COMMENTED** [review]
changed

**Review by shoshone-dev — APPROVED** [review]
/lgtm_dev

**Pull request merged**